### PR TITLE
Add a link to the Gamepads module

### LINKED ISSUE #803: Main spec should link to Gamepad module
It would be helpful to have the main spec link to the WebXR Gamepad module spec documentation, probably in the XRInputSource section: https://immersive-web.github.io/webxr/#xrinputsource-interface

Otherwise, people reading the main spec might be unaware that there can potentially be Gamepads on the XRInputSource objects. Even if they are aware of this, the official documentation is difficult to find.

### PR DESCRIPTION
**Pull request by toji**
/fixes #803

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1465,6 +1465,8 @@ interface XRInputSource {
 };
 </pre>
 
+Note: The {{XRInputSource}} interface is also extended by the <a href="https://immersive-web.github.io/webxr-gamepads-module/">WebXR Gamepads Module</a>
```
Comment (Manishearth) [inline index.bs]: bikeshed lets you do ``[[!webxr-gamepads-1]]`` or ``[[!webxr-gamepads-1|WebXR Gamepads Module]]`

Comment (Manishearth) [reply index.bs]: I think this makes it work better in the references section

Comment (toji) [reply index.bs]: Ah, good to know! It appears that the gamepads module (who's identifier should be `webxr-gamepads-module-1`, FWIW) isn't known to bikeshed yet, though.

Comment (Manishearth) [reply index.bs]: I'm seeing it in my local bikeshed data, going to poke at this a bit

Comment (Manishearth) [reply index.bs]: looks like they need to be in biblio-keys.json

Comment (Manishearth) [reply index.bs]: Yeah so it seems like bikeshed is aware of the spec but not aware _enough_ to generate proper bibliography entries. Pinged Tab on the issue, they'll probably know why

Comment (Manishearth) [reply index.bs]: For now we should merge this

**Pull request merged**